### PR TITLE
chore: Use docker-compose to start minIO and layotto

### LINKED ISSUE #636: use docker-compose to start minIO and layotto
**What would you like to be added**:
We have File API quickstart document showing users how to build and start layotto locally:
https://mosn.io/layotto/#/zh/start/file/minio

And now we can add a docker-compose to start minIO and layotto with one command, and then improve the quickstart.

 Examples are redis&layotto quickstart. I modified those redis related quickstarts showing users how to use docker-compose to start redis and layotto:
https://mosn.io/layotto/#/zh/start/state/start
https://mosn.io/layotto/#/en/start/state/start
image

and:
image

**Why is this needed**:
Make layotto more user friendly

### PR DESCRIPTION
**Pull request by bxiiiiii**
**What this PR does**:
* Use docker-compose to start minIO and layotto

**Which issue(s) this PR fixes**:

Fixes #636 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Review by seeflood — CHANGES_REQUESTED** [review]
Looks good. Could you also modify the minio quickstart doc in https://mosn.io/layotto/#/zh/start/file/minio

You can take the [quickstart for state api](https://mosn.io/layotto/#/zh/start/state/start) as an example

If you don't know how to contribute to the doc, check the doc contribution guide https://mosn.io/layotto/#/zh/development/contributing-doc

**Comment by bxiiiiii** [comment]
ok, let me have a try.

File: docker/layotto-file/config_file.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
+{
+    "servers": [
```
Comment (seeflood) [inline docker/layotto-file/config_file.json:2]: How about naming the path as `docker/layotto-minio/config_minio.json`, which is consistent with the style of `docker/layotto-redis/config_redis.json`

**Review by seeflood — APPROVED** [review]

**Review by zhenjunMa — APPROVED** [review]

**Comment by seeflood** [comment]
Thanks for your contribution and welcome to the community !

**Pull request merged**